src/mlang/utils/pos.mli

```diff
@@ -18,7 +18,9 @@
 
 (** {2 Source code position} *)
 
-type t
+type t = { pos_filename : string; pos_loc : Lexing.position * Lexing.position }
+(* opened this type. Not very satisfactory but necessary to enforce the
```

Comment:
Instead of opening the type you can just use the `make_position` function below in `test_interpreter.ml/convert_pos` :)